src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ConsentOrderApprovedControllerTest.java

```diff
@@ -110,16 +110,81 @@ public void consentOrderApprovedSuccess() throws Exception {
         result.andExpect(status().isOk());
         assertLetter(result);
         assertConsentOrder(result);
+        assertPensionDocs(result);
+    }
+
+    @Test
+    public void consentOrderApprovedSuccessForPaperApplication() throws Exception {
```

Comment:
I think tests that count on feature-toggles should be in a different class.
Here's how you can change the property value you need: https://www.baeldung.com/spring-tests-override-properties#springBootTest

Reply:
Our tests should test both version (toggled on or off). This way, we're only testing one version.

Reply:
Adjusted tests to make it clear what they're supposed to be testing